test/aggregators/aggregators.js

```diff
@@ -47,6 +47,14 @@ describe('Aggregators', () => {
     });
   });
 
+  it('has no aggregators failing when shortName is called', () => {
+    return walkTree.then(aggregators => {
+      aggregators.forEach(aggregator => {
+        assert.doesNotThrow(_ => aggregator.shortName);
```

Comment:
ah we should use .ok
https://nodejs.org/api/assert.html#assert_assert_ok_value_message

or doesNotEqual undefined then